src/web/install_dependencies.sh

```diff
@@ -1,45 +1,11 @@
 #!/bin/bash
 
-# Install dependencies to current directory, or command-line parameter
-if [[ -z $1 ]]; then
-    NODE_MODULES_DIR="./"
-else
-    NODE_MODULES_DIR=$1
-fi
+# Where is your package.json?
+PACKAGE_JSON_DIR=$1
 
-echo 🟢 Installing npm dependencies to: $NODE_MODULES_DIR
+echo 🟢 Installing npm dependencies in $(pwd) \(sorry, ts-loader needs them there\)
```

Comment:
$(pwd) is the current working directory, not where the script is located (which is what I think you want). $(dirname "$0") is usually sufficient for that.

Reply:
Ah yes, this comment is actually obsolete and should be reverted because it's actually being passed the path via argv[1] again.  Before I was only calling it from the pwd, so I used that, but I like the simple way of getting the script's directory.

This should all be obsolete when it's incorporated into the CMakeLists